distribution.md

```diff
@@ -155,3 +157,88 @@ Where:
     * contain an array of deprecated APIs:
       * should only contain `0.x` or major versions
       * should only contain APIs that correspond to a spec release
+
+### Builder
+
+The following specifies the artifact format for a buildpacks builder.
+
+A Builder is an OCI Image that provides a distributable build environment.
+
+A platform supporting the builder SHOULD allow users to configure the build environment with a provided builder.
+
+#### General Requirements
+
+The Builder image MUST contain an implementation of the [lifecycle](#lifecycle), and [build-time](#build-image) environment and MAY contain [buildpacks](#buildpackage). Platforms SHOULD use builders to ease the build process. 
+
+#### Environment Variables
+
+A Builder MUST be an extension of a Build Image, and MUST retain all the specified environment variables set on the original build image, as specified in the Build Image specifications.
+
+The following environment variables MUST be set on the builder (through the image config's `Env` field):
+
+| Env Variable             | Description                                                                       | Default
+| --------                 | --------                                                                          |-----------
+| `CNB_APP_DIR`            | Application directory of the build environment                                    | `/workspace`
+| `CNB_LAYERS_DIR`         | The directory to create and store `layers` in the build environment               | `/layers`
+| `CNB_PLATFORM_DIR`       | The directory to create and store platform focused files in the build environment | `/platform`
```

Comment:
I'm curious why there are "defaults" if these are required environment variables.

EDIT: Seems like @ekcasey had the same thoughts in [this comment](https://github.com/buildpacks/spec/pull/193/files#r635266996) on the original source.

I propose we change it to MAY instead of MUST. If we make that change, it could be merged with the table below.

Reply:
Ahh yes, we can change it to be MUST.